README.org

```diff
@@ -45,6 +45,7 @@ Feel free to make pull requests to add your images! just keep in mind..
 | [[https://github.com/eccentric-j][eccentric-j]]     | [[./splashes/doom/cute-demon.png]]                 | [[https://github.com/eccentric-j/doom-icon][→ Link]] |
 | [[https://github.com/hugonobrega][hugonobrega]]     | [[./splashes/doom/doom-emacs-slant-out-color.png]] |        |
 | [[https://github.com/hugonobrega][hugonobrega]]     | [[./splashes/doom/doom-emacs-slant-out-bw.png]]    |        |
+| [[https://github.com/abrahamparayil][abrahamparayil]]  | [[./splashes/doom/M-x_butterfly.png][./splashes/doom/M-x_butterfly.png]]              |        |
```

Comment:
Can you put this under emacs? the doom category is for splashes related more to the doom themes in doom emacs (cacodemons, doom guy and so on)